### PR TITLE
Nate/feature/LOOP-1113/status bar reorder with style

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1113

@ps2 you have already seen most of this code.

### CODE REVIEW

File: LoopUI/Views/CGMStatusHUDView.swift [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        super.dismissAlert()
+        
+        statusStackView.addArrangedSubview(glucoseValueHUD)
+        statusStackView.addArrangedSubview(glucoseTrendHUD)
+        glucoseValueHUD.isHidden = false
+        glucoseTrendHUD.isHidden = false
+    }
+    
+    public func setGlucoseQuantity(_ glucoseQuantity: Double, at glucoseStartDate: Date, unit: HKUnit, staleGlucoseAge: TimeInterval, sensor: SensorDisplayable?) {
+        var accessibilityStrings = [String]()
```
Comment (rickpasetto) [inline LoopUI/Views/CGMStatusHUDView.swift:89]: This feels like it should be a function in a `ViewModel`, and put under unit test.

File: LoopUI/Views/CGMStatusHUDView.swift [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        } else {
+            sensorAlertState = .ok
+        }
+        glucoseValueHUD.unitLabel.text = unit.localizedShortUnitString
+        accessibilityValue = accessibilityStrings.joined(separator: ", ")
+    }
+    
+    private lazy var timeFormatter = DateFormatter(timeStyle: .short)
+    
+    private var stalenessTimer: Timer?
```
Comment (rickpasetto) [inline LoopUI/Views/CGMStatusHUDView.swift:129]: Seems to me `stalenessTimer`, and all of the associated variables/functions below should live in a `ViewModel`.

**Review by rickpasetto — CHANGES_REQUESTED** [review]
Added a suggestion to move some code in `CGMStatusHUDVIew.swift` into a `ViewModel` and put under unit test.  Other than that, this looks really, really good, @nhamming... Nice job!

General comment: `StatusTableViewController` is ripe for breaking into separate files, IMHO; it is becoming a bit of a dumping ground for a lot of stuff.  I'm not suggesting doing this for this PR, but just worth commenting on.

File: LoopUI/Views/CGMStatusHUDView.swift (hunk 1 above)
Comment (nhamming) [reply LoopUI/Views/CGMStatusHUDView.swift:89]: I'll include a comment and handle this with the LOOP-1293, which is the CGM specific ticket for the status bar.

File: LoopUI/Views/CGMStatusHUDView.swift (hunk 2 above)
Comment (nhamming) [reply LoopUI/Views/CGMStatusHUDView.swift:129]: I'll include a comment and handle this with the LOOP-1293, which is the CGM specific ticket for the status bar.

**Review by rickpasetto — APPROVED** [review]
@nhamming I'm fine with making the changes in a later PR.  Approved.  👍

File: Loop/View Controllers/StatusTableViewController.swift [hunk 3]
```diff
@@ -1309,6 +1323,36 @@ final class StatusTableViewController: ChartsTableViewController {
...
     }
+    
+    @objc private func pumpStatusTapped( _ sender: UIGestureRecognizer) {
+        if let pumpManagerUI = deviceManager.pumpManager {
+            var completionNotifyingVC = pumpManagerUI.settingsViewController()
+            completionNotifyingVC.completionDelegate = self
+            self.present(completionNotifyingVC, animated: true, completion: nil)
+        } else {
+            // currently only one pump manager is supported, so start that setup workflow
+            if let pumpManager = deviceManager.availablePumpManagers.first,
```
Comment (ps2) [inline Loop/View Controllers/StatusTableViewController.swift]: We should show a selector here if there are multiple available.

**Review by ps2 — CHANGES_REQUESTED** [review]

**Review by ps2 — APPROVED** [review]
LGTM!

File: Loop/View Controllers/StatusTableViewController.swift [hunk 4]
```diff
@@ -1330,12 +1330,34 @@ final class StatusTableViewController: ChartsTableViewController {
...
+        let pumpManagers = deviceManager.availablePumpManagers
+        
+        switch pumpManagers.count {
+        case 1:
+            if let pumpManager = pumpManagers.first,
                 let pumpManagerType = deviceManager.pumpManagerTypeByIdentifier(pumpManager.identifier)
             {
                 setupPumpManager(for: pumpManagerType)
             }
+        case let x where x > 1:
```
Comment (rickpasetto) [inline Loop/View Controllers/StatusTableViewController.swift]: [nit] Why not just `default`?

**Review by rickpasetto — APPROVED** [review]
Just one [nit] but other than that LGTM 🐿️

**Pull request merged**